modules/apps/mentions/mentions-api/src/main/resources/content/Language.properties

```diff
@@ -2,4 +2,4 @@ asset-entry-mention-email-body=Asset Entry Mention Email Body
 asset-entry-mention-email-subject=Asset Entry Mention Email Subject
 comment-mention-email-body=Comment Mention Email Body
 comment-mention-email-subject=Comment Mention Email Subject
-mentions-group-service-configuration-name=Mentions Service
\ No newline at end of file
+mentions-group-service-configuration-name=Mentions
```

Comment:
Don't we want this to be `Service` instead?  Like we're doing to Knowledge Base above?